README.md

```diff
@@ -1,7 +1,7 @@
 terser
 ======
 
-![Terser](https://raw.githubusercontent.com/terser-js/terser/master/logo.png)
+![Terser](https://raw.githubusercontent.com/terser-js/terser/master/logo.svg)
```

Comment:
@kkocdko Cc: @fabiosantoscode

Re: https://github.com/terser-js/terser/pull/415#issuecomment-521357450

Needs additional QSP for GH of `?sanitize=true` appended to display in README.md. Tis a GH rule for showing SVGs. :)

Reply:
Btw if you don't constrain it with a HTML `img` tag using  `width` and `height` it's going to fill the width and be rather large. You can of course size it  in the SVG itself *(usually CSS)* as well but that sort of defeats some of the purpose.